lumicks/pylake/tests/test_imaging_confocal/test_scan.py

```diff
@@ -328,11 +328,11 @@ def test_single_frame_times(dim_x, dim_y, line_padding, start, dt, samples_per_p
     [
         (5, 6, 3, 3, 14, 4, 4),
         (3, 4, 4, 60, 1592916040906356300, 12800, 30),
-        (3, 2, 2, 60, 1592916040906356300, 12800, 3000),
+        (3, 2, 3, 60, 1592916040906356300, 12800, 3000),
     ],
 )
 def test_multiple_frame_times(dim_x, dim_y, frames, line_padding, start, dt, samples_per_pixel):
-    img = np.ones((dim_x, dim_y, frames))
+    img = np.ones((frames, dim_x, dim_y))
```

Comment:
hmm somewhat concerning that this passed even with an incorrect shape definition before

Reply:
Indeed. Was a flawed test. Never tested against the actually requested frame count. Fixed now though. It'll fire if you change it back to the error.